### PR TITLE
Add pds-scheduler-themes CLI tool for automated release theme creation

### LINKED ISSUE #41: As a user, I want to automate creation of release theme issues from CSV schedules
## Checked for duplicates
Yes - I've already checked

## 🚀 Describe the feature request
Create a CLI tool that automates the creation of release theme issues from CSV schedule files.

## 👤 User persona
Release Manager, Engineering Operations Lead

## 💪 Motivation
A subset of release themes are recurring planning, level-of-effort, and I&T tasks that repeat from build to build. Manually creating 10-15 themed issues across multiple repositories for each build cycle is time-consuming and error-prone. Automating this process from a CSV schedule ensures consistency, saves time, and reduces the risk of missing critical release planning tasks.

### Requirements

**Input:**
- CSV file with columns: Title, Repo, Start Date, End Date, Description, Checklist (semicolon-delimited), GitHub Project Product
- Build number (e.g., 17 for "B17")

**Behavior:**
- Prepend build number to issue titles (e.g., "B17 Release Planning")
- Check if issue already exists in repository and skip if found (prevent duplicates)
- Create issues with appropriate labels: `theme`, `Epic`, `i&t.skip`, and build label (e.g., `B17`)
- Parse checklist items into markdown checkbox format
- Add issues to NASA-PDS/6 project and build-specific GitHub Projects
- Support dry-run mode to preview without creating issues

**Output:**
- Clear summary showing created, skipped (duplicates), and failed issues
- Issue URLs for successfully created issues

**CSV Date Format:**
- Accept MM/DD/YY format (e.g., "9/5/25")

**Nice to Have:**
- Set GitHub Project custom fields (Product, Start Date, End Date)
- Create build labels if they don't exist in target repositories

**Additional Context:**
This supports B17+ release cycles where release planning involves recurring themes across repositories like validate, registry-api, pds4-information-model, and system-i-n-t.

---
## ⚙️ Engineering Details
[To be filled by engineering team]

## 🎉 Integration & Test
[To be filled by engineering team]

### PR DESCRIPTION
**Pull request by jordanpadams**
## Summary
Implements a new command-line tool that automates the creation of release theme issues from CSV schedule files. This tool streamlines the process of creating recurring planning, level-of-effort, and I&T tasks for each build cycle.

## Features
- **CSV-driven issue creation**: Reads theme schedules from CSV files containing title, repository, dates, description, checklist, and product metadata
- **Build labeling**: Automatically prefixes issue titles with build number (e.g., "B17 Release Planning")
- **Duplicate detection**: Checks for existing issues with the same title and skips creation to prevent duplicates  
- **Label management**: Applies theme, Epic, i&t.skip labels and creates build-specific labels (e.g., B17) if they don't exist
- **Project integration**: Adds issues to build-specific GitHub Projects and sets custom fields (Product, Start date, End date) via GraphQL
- **Checklist support**: Parses semicolon-delimited checklist items from CSV and formats them as markdown checkboxes
- **Dry-run mode**: Preview what would be created without actually creating issues
- **Comprehensive test coverage**: 26 unit tests covering all functions and edge cases

## Usage
```bash
# Preview issues (dry-run)
pds-scheduler-themes --csv-file schedule.csv --build-number 17 --dry-run

# Create issues
pds-scheduler-themes --csv-file schedule.csv --build-number 17

# With debug logging
pds-scheduler-themes --csv-file schedule.csv --build-number 17 --debug
```

## Testing
* Automated unit tests added with code updates
* I can confirm this worked for [B17](https://github.com/orgs/NASA-PDS/projects/22/views/1) 🎉 

Screenshot 2026-01-10 at 9 34 30 AM

## Related Issues
Closes #41

## Additional Notes
- Uses GitHub CLI (gh) for all GitHub operations
- Requires gh to be installed and authenticated
- Example CSV file included in examples/b17_schedule_prep_spreadsheet.csv
- Full documentation added to README.md

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -118,7 +119,7 @@ where = src
 #
 # See https://docs.pytest.org/ for more information.
 [tool:pytest]
-addopts = -n auto --cov=pds
+addopts = -n auto --cov=src
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply setup.cfg]: `--cov-pds` does't work and is wrong. that is now correct.

File: README.md [hunk 2]
```diff
@@ -25,9 +26,54 @@ Install with:
...
+# With debug logging
+pds-scheduler-themes --csv-file schedule.csv --build-number 17 --debug
+```
+
+**CSV Format:**
+
+The CSV file should contain these columns:
+- **Title**: Issue title (will be prefixed with build number, e.g., "B17 Release Planning")
+- **Repo**: Repository name in format `NASA-PDS/repo-name`
+- **Start Date**: Start date in MM/DD/YY format (e.g., "9/5/25")
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply README.md]: updated
